Source/WebKit/UIProcess/API/Cocoa/WKWebViewPrivate.h

```diff
@@ -268,6 +268,8 @@ for this property.
 
 - (void)_doAfterNextVisibleContentRectUpdate:(void (^)(void))updateBlock WK_API_AVAILABLE(macos(13.0), ios(16.0));
 
+- (void)_generateSyntheticCommandP:(void (^)(BOOL))completionBlock;
```

Comment:
This should be marked as `WK_API_AVAILABLE(macOS(WK_MAC_TBA), iOS(WK_IOS_TBA))`

Reply:
do the `TBA`s mean like a future release? If that's the case, will that affect the Safari PR's mergability?